packages/bpk-component-datatable/src/utils.js

```diff
@@ -17,10 +17,21 @@
  */
 /* @flow strict */
 
-// TODO: Remove these functions once we want to change the API to match the react-table library API
+const ROOT_FONT_SIZE_PX = 16;
+
+export const pxToRem = (value: string | number) => {
+  let parsed = value;
+
+  if (typeof value === 'number') {
```

Comment:
What happens if someone doesn't pass a number and actually parses the string `16px` would this attempt to just parse as is?

Reply:
Hmm yes, it wouldn't be parsed to `rem` if someone passes a string. Let me see, I'll change this slightly